.travis.yml

```diff
@@ -4,65 +4,113 @@ os:
   - linux
   - osx
 
-compiler: 
-    - gcc
-    - clang
+compiler:
+  - gcc
+  - clang
 
 env:
-    - EASTL_CONFIG=Debug
-    - EASTL_CONFIG=Release
+  - EASTL_CONFIG=Debug
+  - EASTL_CONFIG=Release
 
 addons:
   apt:
     sources:
-    - ubuntu-toolchain-r-test
+      - ubuntu-toolchain-r-test
     packages:
-    - gcc-4.8
-    - g++-4.8
-    - clang
+      - gcc-4.8
+      - g++-4.8
+      - clang
 
 matrix:
     include:
         - compiler: clang "debug build with clang3.9"
           env: EASTL_CONFIG=Debug   USE_CLANG_39=yes
           os: osx
-        - compiler: clang  "release build with clang3.9"
+        - compiler: clang "release build with clang3.9"
           env: EASTL_CONFIG=Release USE_CLANG_39=yes
           os: osx
+        - compiler: clang "build with Buck"
+          env: USE_BUCK=yes
```

Comment:
It currently only does a release build. Buck does not have built-in release and debug targets like CMake does, however they are [easy to implement](https://github.com/njlr/buck-custom-flavours/blob/master/.buckconfig). 

Is this a requirement? Happy to add it to the PR.